modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-migration_{context}"]
+= Migrating to the OVN-Kubernetes network provider
+
+As a cluster administrator, you can upgrade the Pod network provider for your cluster to the OVN-Kubernetes Pod network provider.
+During the migration, you must reboot every node in your cluster.
+
+[IMPORTANT]
+====
+While performing the migration, your cluster is unavailable and workloads may be interrupted.
+Perform the migration only when an interruption in service is acceptable.
+====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+* A cluster installed on bare metal infrastructure configured with the OpenShift SDN Pod network provider.
+* The cluster is in a known good state, without any errors.
+
+.Procedure
+
+. To backup the configuration for the cluster network, enter the following command:
```

Comment:
Are we trying to get away from `enter the following command` and similar?
I would just use `backup the configuration for the cluster network:`